docs/tutorial/get-started.rst

```diff
@@ -133,21 +133,10 @@ You now need to create a Kubeflow model for the Juju controller as follows:
 Deploy CKF
 ----------
 
-MicroK8s uses ``inotify`` to interact with the file system. 
-Kubeflow requires increasing the ``inotify`` limits. 
-To do so, run the following commands:
-
-.. code-block:: bash
-
-    sudo sysctl fs.inotify.max_user_instances=1280
-    sudo sysctl fs.inotify.max_user_watches=655360
-
-If you want these commands to persist across machine restarts, add the following lines to ``/etc/sysctl.conf``:
-
-.. code-block:: bash
-
-    fs.inotify.max_user_instances=1280
-    fs.inotify.max_user_watches=655360
+> Older version of MicroK8s were setting low `inotify` limits, that did not comply
```

Comment:
issue: this is not rendered properly - from the branch's preview:

<img width="1617" height="483" alt="Image" src="https://github.com/user-attachments/assets/1f5f632b-0f58-49b6-b4e1-d5b7a877601c" />